Builds/containers/shared/update-rippled.sh

```diff
@@ -2,6 +2,13 @@
 
 # auto-update script for rippled daemon
 
+# Check for sudo/root permissions
+if [ `id -u` -ne 0 ]; then
```

Comment:
it's a nit-pick, but other places here use `$(cmd)` for subshells, so I guess I would prefer that strictly for consistency (I don't think there is any practical difference in this case, but let me know if I'm wrong...)

Reply:
No issues. I can change it for consistency.